webapp/containers/App/App.js

```diff
@@ -123,7 +125,4 @@ const mapDispatchToProps = dispatch => {
   );
 };
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(App, 'App');
+export default connect(mapStateToProps, mapDispatchToProps)(App, 'App');
```

Comment:
I think the linter wanted this split up. Did you run prettier on this file and it put it back?